src/Applications/GEOSdas_App/GEOSdas.csm

```diff
@@ -3189,53 +3198,100 @@ endif
 # ------------------------------
   Sub LandAnalysisRun_() 
 # ------------------------------
+
+#sqz--merge--start
+
   if ( $?ECHO___ ) set echo
 
-  cd $FVWORK
+  if ( ! $LDAS_ANA ) exit 0
+
+    cd $FVWORK
+#   link $FVWORK for ldas met_forcing  access 
+       /bin/rm -f $FVHOME/lana/forc
+       /bin/ln -s $FVWORK $FVHOME/lana/forc
+
   set rstdate = ( `rst_date ./d_rst` )
-  set mydate = (`tick $rstdate -21600`)  # tick back 6 hours
-
-  set DURATION = "060000"
-  set SUBDIR = "ldas.$mydate[1].$mydate[2]"
-  set LDINPUT_FORCE = "$FVWORK"
-  set EXPDOMAIN = "GLOBAL"
-  set RESOLUTION = "144x91"
-  set THREEHOURS = 10800
-
-  ldsetup -expid $EXPID \
-          -nymd $mydate[1] \
-          -nhms $mydate[2] \
-          -duration $DURATION \
-          -fvroot $FVROOT \
-          -fvhome $FVHOME \
-          -fvwork $FVWORK \
-          -fvwork_subdir $SUBDIR \
-          -expdomain $EXPDOMAIN \
-          -resolution $RESOLUTION \
-          -ldinput_force $FVWORK \
-          -mettag $EXPID \
-          -forcedtstep $THREEHOURS \
-          -rstrtdir $FVWORK \
-          -rstrtid $EXPID \
-          -gid $GID \
-          -noprompt
-
-  echo y | lenkf.pl
-
-# Move output to top FVWORK directory
-# -----------------------------------
-  cd $FVWORK
-  foreach file ( `find $SUBDIR -name $EXPID\*.txt`)
-     /bin/mv $file .
-  end
-  foreach file ( `find $SUBDIR -name $EXPID\.ens\*`)
-     /bin/mv $file .
-  end
+  set enddate = (`tick $rstdate 21600`)
+
+# go to LDHOME to run ldas
+      cd $LDHOME/run
+# compare the starting time
+      echo "ldas_home_dir: ", $LDHOME
+      set lcapdat = `cat cap_restart | cut -c1-8`
+      set lcaptim = `cat cap_restart | cut -c10-15`
+      echo "ldas_6h_window starting at: ", $lcapdat, $lcaptim
+      echo "adas_anal_window starting at: ", $rstdate[1], $rstdate[2]
+
+# submit job and capture job ID 
+
+       set jobldas = "lenkf.j"
+       set jobIDlong = `$PBS_BIN/sbatch $jobldas`
+       set jobID = `echo $jobIDlong  |awk -F'[ ]' '{print $4}'`
+       cd -
+       setenv ldasJobIDs  $jobID
+       echo $ldasJobIDs ": LDAS coupling lenkf jobID in LandAnalysisRun"
+       cd $FVWORK
 
                               # Last line of LandAnalysisRun_()
 \end
 
+#sqz---merge--end
+#.............................................................................
+
+## sqz--merge--start
+#------------------------
+  Sub StageLdasIncr_()
+#-----------------------
+   if ( $?ECHO___ ) set echo
+
+    cd $FVWORK
+
+    set LINC_DIR=$LDHOME/output/*/cat/ens_avg/
+
+    # default: 
+    set ldas_int = 10800
```

Comment:
Should we really hard-wire the LDAS and ADAS time steps here?  Do these numbers exist in some variables somewhere?